src/conn/conn_cache_pool.c

```diff
@@ -506,7 +542,14 @@ __cache_pool_adjust(WT_SESSION_IMPL *session,
 		reserved = cache->cp_reserved;
 		adjusted = 0;
 
-		read_pressure = cache->cp_current_read / highest;
+		/*
+		 * The read pressure is calculated as a percentage of how
+		 * much read pressure there is on this participant compared
+		 * to the participant with the most activity. The closer we
+		 * are to the most active the more cache we should get
+		 * assigned.
+		 */
+		read_pressure = cache->cp_pass_read / highest_percentile;
```

Comment:
Can you talk me through this part because I'm a little confused.

We're taking into account reads, eviction and application waits to calculate which participants could most benefit from more cache.  But here we're only using reads to calculate the "pressure"?  I'd expected that we would combine the scores and use the combined value everywhere (i.e. `cp_pass_pressure` instead of `cp_pass_read`).